common/djangoapps/external_auth/backends.py

```diff
@@ -127,6 +127,10 @@ def authenticate(self, ticket, service):
            NB: Use of PT to identify proxy
         """
         username, authentication_response = _verify(ticket, service)
+
+        first_name = authentication_response[0][1].find(CAS + 'first_name').text
```

Comment:
Also is the response guaranteed to have elements [0] and [1] and so on, or should it be enclose in `try..except`?